test/acceptance/features/volumeMount.feature

```diff
@@ -0,0 +1,166 @@
+Feature: Bind an application to a service using volume mount
```

Comment:
volume mount is just technical detail here, for user is more important to say that "bindings get injected as files in application container`. Please rephrase.

Reply:
Done. 172198aa20026266b5b017693826e754be0f1567